integration-test/src/test/java/org/bitcoinj/core/PeerTest.java

```diff
@@ -514,17 +515,17 @@ public void pingPong() throws Exception {
         inbound(writeTarget, new Pong(pingMsg.getNonce()));
         pingAndWait(writeTarget);
         assertTrue(future.isDone());
-        long elapsed = future.get();
-        assertTrue("" + elapsed, elapsed > 1000);
-        assertEquals(elapsed, peer.getLastPingTime());
-        assertEquals(elapsed, peer.getPingTime());
+        Duration elapsed = future.get();
+        assertTrue("" + elapsed, elapsed.toMillis() > 1000);
```

Comment:
I propose to change the first argument to

```
elapsed.toMilllis() + " ms"
```

Duration.toString() unfortunately isn't very human readable.

Reply:
Fixed and force-pushed.